scripts/fast-start.sh

```diff
@@ -181,6 +189,9 @@ KEYNAME=$(grep keyname /root/AppScalefile | cut -f 2 -d ":")
 [ -z "${KEYNAME}" ] && { echo "Cannot discover keyname: is AppScale deployed?" ; exit 1 ; }
 
 # Deploy sample app.
+if [ -z "${ADMIN_EMAIL}" ]; then
```

Comment:
Should we check if this is a docker container instead of if the admin email is set?
Seems more appropriate.